numba/core/errors.py

```diff
@@ -544,7 +544,8 @@ class NotDefinedError(IRError):
 
     def __init__(self, name, loc=None):
         self.name = name
-        msg = "Variable '%s' is not defined." % name
+        msg = "The compiler failed to analyze the bytecode\n \
+               Variable '%s' is not defined." % name
```

Comment:
```suggestion
        msg = ("The compiler failed to analyze the bytecode. "
               "Variable '%s' is not defined." % name)
```

The existing patch produces this:
```
numba.core.errors.NotDefinedError: Failed in nopython mode pipeline (step: analyzing bytecode)
The compiler failed to analyze the bytecode
                Variable '$4setup_with.1' is not defined.
```
the suggestion produces this:
```
numba.core.errors.NotDefinedError: Failed in nopython mode pipeline (step: analyzing bytecode)
The compiler failed to analyze the bytecode. Variable '$4setup_with.1' is not defined.
```

Reply:
Thank you for the suggestion, I've just committed it. Quite a coincidence when I tested it with my terminal window and it didn't look like that.